packages/mdc-select/_mixins.scss

```diff
@@ -34,8 +34,7 @@
 
 @mixin mdc-select-focused-label-color($color, $opacity: .87) {
   &:not(.mdc-select--disabled) {
-    .mdc-select__surface:focus,
-    &.mdc-select--open {
+    .mdc-select__surface:focus ~ {
```

Comment:
Yup it does work. It is selecting any sibling appearing after itself. In this case, it is the label.

Reply:
Oh wait. It's working because the floating-label-color mixin is appending the correct class, which makes the combinator valid.

That's pretty confusing without also looking at the mixin... might want to add a comment.